lib/Runtime/Language/JavascriptConversion.cpp

```diff
@@ -538,27 +538,25 @@ namespace Js
         }
 
         // If exoticToPrim is not undefined, then
-        if (nullptr != exoticToPrim)
+        Assert(nullptr != exoticToPrim);
```

Comment:
Should this be a fail-fast? What guarantees that this is always not null?

Reply:
If this is null then result is nullptr and it's not valid to pass nullptr to GetTypeId (prefast took issue with this due to some of the annotations I added).

If we get to this point, we will always have called `JavascriptOperators::GetPropertyReference` for `varMethod`, and we always assign `exoticToPrim` = `varMethod`.